sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBaseParser.g4

```diff
@@ -40,6 +40,11 @@ options { tokenVocab = SqlBaseLexer; }
    * When true, double quoted literals are identifiers rather than STRINGs.
    */
   public boolean double_quoted_identifiers = false;
+
+  /**
+   * When true, identifiers that begin from `:` are considered as named parameters.
+   */
+  public boolean parameters_enabled = false;
```

Comment:
This is a new SQL syntax, it won't break any existing queries. Do we really need a feature flag?

Reply:
I have removed the SQL config.